content/sensu-go/5.20/reference/agent.md

```diff
@@ -790,6 +791,21 @@ See the [example agent configuration file][5] (also provided with Sensu packages
 
 ### General configuration flags
 
```

Comment:
Nikki, this allow-list content is not actually new. I just moved it out of the way because it was where the discover-processes table should be.